### ISSUE TITLE
StackBlitz adds dependencies of NPM packages to my package.json

### ISSUE THREAD

**Issue by jhenderson2099** [stackblitz/core#956]
When I add an NPM package to my Angular project, it appears that StackBlitz adds the dependencies of that package (i.e., the `dependencies` within the NPM package's `package.json`) as `dependencies` within my project's `package.json`.  

This appears to be incorrect because my project doesn't directly use these packages.

In a local VS Code project, these packages would be installed under the `node_modules` branch for the NPM package that my project uses.

Is this behavior by design?  If so, it seems that you could end up pulling in a number of NPM packages that aren't used (directly) by the project.

**Comment by willi84** [comment]
it sounds like the different beha behavior of --save-dev or installing globally you want

 I guess you need to give a good example, because otherwise, it is hard to follow, why the behavior should be different. If my angular project needs sth. to run, then for sure, it should be added to package.json

or why you need to do a npm install without "save|save-dev" or a npm install -g ?

**Comment by purplem1lk** [comment]
Thank you for your patience - We are unable to provide assistance without further information about the issue you are seeing and will be closing this accordingly. If your issue persists, feel free to tag me or ask to reopen this issue with a working project link, description, and screenshots of the error. Thank you! :)
